src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -130,6 +137,13 @@ public override async Task<int> ExecuteAsync()
             }
         }
 
+        private async Task<IEnumerable<DependencyDetail>> GetBuildDependenciesAsync(Build build)
```

Comment:
Any reason to not use GetDependenciesAsync in DarcLib's DependencyGraph class?

Reply:
am confused, isn't that what the call to  `remote.GetDependenciesAsync(build.GitHubRepository, build.Commit);` does?

Reply:
So that's going to call, IRemote's GetDependenciesAsync. I was mentioning https://github.com/dotnet/arcade-services/blob/master/src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DependencyGraph.cs#L834

Reply:
The one in the DependencyGraph is specific to building the dependency graph (it's private since it has some some oddities). I think using the remote here is correct.

Reply:
Oh, I see that that function is only an internal DependencyGraph function now. This looks good to m.e